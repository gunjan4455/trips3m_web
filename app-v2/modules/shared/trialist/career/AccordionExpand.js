import React from 'react';
import styled from '@emotion/styled';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';
import Collapsible from 'react-collapsible';

const GAccordianDiv = styled.div({
  '& .Collapsible__trigger': {
    padding: '15px',
    display: 'block',
    position: 'relative',

    '&:after': {
      position: 'absolute',
      content: ' ',
      margin: 'auto',
      width: '8px',
      height: '8px',
      background: 'rgba(0,0,0,0)',
      border: '1px solid #3e3e3e',
      borderTop: '0',
      top: '7px',
      bottom: '10px',
      borderRight: '0',
      right: '12px',
      transform: 'rotate(-45deg)',


    },
  },
  '& .is-open': {
    color: '#0c9a8d',

    '&:after': {
      transform: 'rotate(135deg)',
      top: '14px',
    }
  }
});

const AccordionHiddenBoxChild = styled.div({
  border: '1px solid #e4e4e4',
});

const OpenRolesCount = styled.p({
  top: '18px',
  right: '50px',
});

const ListShowMain = styled.p({
  lineHeight: '26px',
});

const AccordionExpand = ({ list, name }) => {
  return (
    list && list.length && name ?
      <GAccordianDiv className="clearfix relative border at_accordionExpand">
        <div className="clearfix relative">
          <OpenRolesCount className="m0 sfc1 f12 absolute">{list.length} Open Roles</OpenRolesCount>
          <Collapsible trigger={name} className="drop_down">
            <div className="clearfix relative ">
              {
                list.map(job => (
                  <div
                    key={`${name}_${job.id}`}
                    className="clearfix"
                  >
                    <AccordionHiddenBoxChild className="clearfix p15">
                      <div className="col-xs-12 row">
                        <p className="f16a sfc6 fw9 m0 pb8">{job.subCategory}</p>
                        <ListShowMain className="f14 m0">
                          {
                            job.rolesAndResponsibities
                              .replace(/●/g, ' ')
                              .replace(/•/g, ' ')
                          }
                        </ListShowMain>
                      </div>
                      <div className="col-xs-12 p0 pt15">
                        <Link
                          to={`/career/${job.id}`}
                          className="btn-pri wfull fright radius2 pt8 pb8"
                        >
                          View Details
                        </Link>
                      </div>
                    </AccordionHiddenBoxChild>
                  </div>
                ))
              }
            </div>
          </Collapsible>
        </div>
      </GAccordianDiv> : null
  );
};

AccordionExpand.propTypes = {
  list: PropTypes.array.isRequired,
  name: PropTypes.string.isRequired
};

export default AccordionExpand;
