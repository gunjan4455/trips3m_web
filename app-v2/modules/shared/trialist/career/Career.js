import React from 'react';
import PropTypes from 'prop-types';

import AboutText from 'modules/shared/trialist/about/AboutText';
import AccordionExpand from 'modules/shared/trialist/career/AccordionExpand';
import TrialistHeading from 'modules/shared/trialist/TrialistHeading';

const CAREER_LIST = {
  salesAndOperations: 'Sales & Operations',
  development: 'Development',
  technology: 'Development',
  digitalMarketing: 'Digital Marketing'
};

const Career = ({ careers }) => (
  <div>
    <div className="clearfix p24 mb32">
      <div className="mb24 at_heading1">
        <TrialistHeading heading="Solving Holiday Woes" />
      </div>
      <AboutText
        text="Holiday planning is a mix of gathering information from multiple channels,
         talking to several service providers and constantly turning into device to plan
          and pay for small things. Ever thought how relaxed a traveler will be if all
          the answers are provided at one place without any hassle? Welcome to TravelTriangle
        where we are solving real time complex problems and building the biggest holiday
        eco-system."
      />
    </div>

    <div className="clearfix p8 mb32">
      <div className="mb24 at_heading2">
        <TrialistHeading
          heading="Current Openings"
          extraStyle={{ textAlign: 'center' }}
        />
      </div>
      <div className="at_text2">
        {
          careers && Object.keys(careers).map(key => (
            <AccordionExpand
              key={key}
              name={CAREER_LIST[key] || ''}
              list={careers[key]}
            />
          ))
        }
      </div>
    </div>
  </div>
);

Career.propTypes = {
  careers: PropTypes.object.isRequired
};

export default Career;
