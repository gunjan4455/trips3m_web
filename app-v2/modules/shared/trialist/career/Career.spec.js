import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import Career from './Career';
import { careerData } from './test/careerData';
import AccordionExpand from './AccordionExpand';

configure({ adapter: new Adapter() });

const DEFAULT_PROPS = {
  careers: careerData
};

const shallowWrapperCareerMain = (props) => {
  return shallow(<Career {...props} />);
};

describe('career module', () => {
  let props = { ...DEFAULT_PROPS };

  afterEach(() => {
    props = { ...DEFAULT_PROPS };
  });

  it('should render correctly', () => {
    const wrapper = shallowWrapperCareerMain(props);
    expect(wrapper).toMatchSnapshot();
  });

  it('should pass proper name in Acccording Expand component', () => {
    const wrapper = shallowWrapperCareerMain(props);
    expect(wrapper.find(AccordionExpand).at(0).prop('name')).toEqual('Development');
  });

});
