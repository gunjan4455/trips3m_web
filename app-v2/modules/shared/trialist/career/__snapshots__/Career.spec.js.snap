// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`career module should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Career
    careers={
      Object {
        "development": Array [],
        "salesAndOperations": Array [
          Object {
            "age": "",
            "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
            "category": "Sales and Operations",
            "education": "Graduation",
            "experience": "0-3 years",
            "goodToHave": "Experience in sales, operations and vendor management.",
            "howToApply": "career Site",
            "id": 25,
            "location": "Gurgaon",
            "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
            "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
            "subCategory": "​Travel Consultant",
            "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
          },
        ],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="clearfix p24 mb32"
        >
          <div
            className="mb24 at_heading1"
          >
            <TrialistHeading
              extraStyle={Object {}}
              heading="Solving Holiday Woes"
            />
          </div>
          <AboutText
            text="Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system."
          />
        </div>,
        <div
          className="clearfix p8 mb32"
        >
          <div
            className="mb24 at_heading2"
          >
            <TrialistHeading
              extraStyle={
                Object {
                  "textAlign": "center",
                }
              }
              heading="Current Openings"
            />
          </div>
          <div
            className="at_text2"
          >
            <AccordionExpand
              list={Array []}
              name="Development"
            />
            <AccordionExpand
              list={
                Array [
                  Object {
                    "age": "",
                    "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                    "category": "Sales and Operations",
                    "education": "Graduation",
                    "experience": "0-3 years",
                    "goodToHave": "Experience in sales, operations and vendor management.",
                    "howToApply": "career Site",
                    "id": 25,
                    "location": "Gurgaon",
                    "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                    "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                    "subCategory": "​Travel Consultant",
                    "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                  },
                ]
              }
              name="Sales & Operations"
            />
          </div>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="mb24 at_heading1"
            >
              <TrialistHeading
                extraStyle={Object {}}
                heading="Solving Holiday Woes"
              />
            </div>,
            <AboutText
              text="Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system."
            />,
          ],
          "className": "clearfix p24 mb32",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <TrialistHeading
                extraStyle={Object {}}
                heading="Solving Holiday Woes"
              />,
              "className": "mb24 at_heading1",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "extraStyle": Object {},
                "heading": "Solving Holiday Woes",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "text": "Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system.",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="mb24 at_heading2"
            >
              <TrialistHeading
                extraStyle={
                  Object {
                    "textAlign": "center",
                  }
                }
                heading="Current Openings"
              />
            </div>,
            <div
              className="at_text2"
            >
              <AccordionExpand
                list={Array []}
                name="Development"
              />
              <AccordionExpand
                list={
                  Array [
                    Object {
                      "age": "",
                      "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                      "category": "Sales and Operations",
                      "education": "Graduation",
                      "experience": "0-3 years",
                      "goodToHave": "Experience in sales, operations and vendor management.",
                      "howToApply": "career Site",
                      "id": 25,
                      "location": "Gurgaon",
                      "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                      "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                      "subCategory": "​Travel Consultant",
                      "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                    },
                  ]
                }
                name="Sales & Operations"
              />
            </div>,
          ],
          "className": "clearfix p8 mb32",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <TrialistHeading
                extraStyle={
                  Object {
                    "textAlign": "center",
                  }
                }
                heading="Current Openings"
              />,
              "className": "mb24 at_heading2",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "extraStyle": Object {
                  "textAlign": "center",
                },
                "heading": "Current Openings",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <AccordionExpand
                  list={Array []}
                  name="Development"
                />,
                <AccordionExpand
                  list={
                    Array [
                      Object {
                        "age": "",
                        "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                        "category": "Sales and Operations",
                        "education": "Graduation",
                        "experience": "0-3 years",
                        "goodToHave": "Experience in sales, operations and vendor management.",
                        "howToApply": "career Site",
                        "id": 25,
                        "location": "Gurgaon",
                        "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                        "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                        "subCategory": "​Travel Consultant",
                        "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                      },
                    ]
                  }
                  name="Sales & Operations"
                />,
              ],
              "className": "at_text2",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": "development",
                "nodeType": "function",
                "props": Object {
                  "list": Array [],
                  "name": "Development",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "salesAndOperations",
                "nodeType": "function",
                "props": Object {
                  "list": Array [
                    Object {
                      "age": "",
                      "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                      "category": "Sales and Operations",
                      "education": "Graduation",
                      "experience": "0-3 years",
                      "goodToHave": "Experience in sales, operations and vendor management.",
                      "howToApply": "career Site",
                      "id": 25,
                      "location": "Gurgaon",
                      "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                      "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                      "subCategory": "​Travel Consultant",
                      "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                    },
                  ],
                  "name": "Sales & Operations",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="clearfix p24 mb32"
          >
            <div
              className="mb24 at_heading1"
            >
              <TrialistHeading
                extraStyle={Object {}}
                heading="Solving Holiday Woes"
              />
            </div>
            <AboutText
              text="Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system."
            />
          </div>,
          <div
            className="clearfix p8 mb32"
          >
            <div
              className="mb24 at_heading2"
            >
              <TrialistHeading
                extraStyle={
                  Object {
                    "textAlign": "center",
                  }
                }
                heading="Current Openings"
              />
            </div>
            <div
              className="at_text2"
            >
              <AccordionExpand
                list={Array []}
                name="Development"
              />
              <AccordionExpand
                list={
                  Array [
                    Object {
                      "age": "",
                      "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                      "category": "Sales and Operations",
                      "education": "Graduation",
                      "experience": "0-3 years",
                      "goodToHave": "Experience in sales, operations and vendor management.",
                      "howToApply": "career Site",
                      "id": 25,
                      "location": "Gurgaon",
                      "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                      "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                      "subCategory": "​Travel Consultant",
                      "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                    },
                  ]
                }
                name="Sales & Operations"
              />
            </div>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="mb24 at_heading1"
              >
                <TrialistHeading
                  extraStyle={Object {}}
                  heading="Solving Holiday Woes"
                />
              </div>,
              <AboutText
                text="Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system."
              />,
            ],
            "className": "clearfix p24 mb32",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <TrialistHeading
                  extraStyle={Object {}}
                  heading="Solving Holiday Woes"
                />,
                "className": "mb24 at_heading1",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "extraStyle": Object {},
                  "heading": "Solving Holiday Woes",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "text": "Holiday planning is a mix of gathering information from multiple channels, talking to several service providers and constantly turning into device to plan and pay for small things. Ever thought how relaxed a traveler will be if all the answers are provided at one place without any hassle? Welcome to TravelTriangle where we are solving real time complex problems and building the biggest holiday eco-system.",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="mb24 at_heading2"
              >
                <TrialistHeading
                  extraStyle={
                    Object {
                      "textAlign": "center",
                    }
                  }
                  heading="Current Openings"
                />
              </div>,
              <div
                className="at_text2"
              >
                <AccordionExpand
                  list={Array []}
                  name="Development"
                />
                <AccordionExpand
                  list={
                    Array [
                      Object {
                        "age": "",
                        "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                        "category": "Sales and Operations",
                        "education": "Graduation",
                        "experience": "0-3 years",
                        "goodToHave": "Experience in sales, operations and vendor management.",
                        "howToApply": "career Site",
                        "id": 25,
                        "location": "Gurgaon",
                        "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                        "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                        "subCategory": "​Travel Consultant",
                        "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                      },
                    ]
                  }
                  name="Sales & Operations"
                />
              </div>,
            ],
            "className": "clearfix p8 mb32",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <TrialistHeading
                  extraStyle={
                    Object {
                      "textAlign": "center",
                    }
                  }
                  heading="Current Openings"
                />,
                "className": "mb24 at_heading2",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "extraStyle": Object {
                    "textAlign": "center",
                  },
                  "heading": "Current Openings",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <AccordionExpand
                    list={Array []}
                    name="Development"
                  />,
                  <AccordionExpand
                    list={
                      Array [
                        Object {
                          "age": "",
                          "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                          "category": "Sales and Operations",
                          "education": "Graduation",
                          "experience": "0-3 years",
                          "goodToHave": "Experience in sales, operations and vendor management.",
                          "howToApply": "career Site",
                          "id": 25,
                          "location": "Gurgaon",
                          "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                          "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                          "subCategory": "​Travel Consultant",
                          "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                        },
                      ]
                    }
                    name="Sales & Operations"
                  />,
                ],
                "className": "at_text2",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": "development",
                  "nodeType": "function",
                  "props": Object {
                    "list": Array [],
                    "name": "Development",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": "salesAndOperations",
                  "nodeType": "function",
                  "props": Object {
                    "list": Array [
                      Object {
                        "age": "",
                        "careerImageUrl": "https://d2ox9n3d2kcupy.cloudfront.net/attachments/careers/25/original/travel-clipart-travel_063.gif",
                        "category": "Sales and Operations",
                        "education": "Graduation",
                        "experience": "0-3 years",
                        "goodToHave": "Experience in sales, operations and vendor management.",
                        "howToApply": "career Site",
                        "id": 25,
                        "location": "Gurgaon",
                        "rolesAndResponsibities": "1. Maintaining relationship with travelers by calling and understanding their trip related details.",
                        "skills": "​• Freshers are welcome.
​• 0 to 2 years of experience in sales, operations, vendor management",
                        "subCategory": "​Travel Consultant",
                        "whatYouGet": "1. Open Culture and Great learning curve.
2. Ample opportunity for career growth.",
                      },
                    ],
                    "name": "Sales & Operations",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
